nautobot_design_builder/signals.py

```diff
@@ -67,7 +67,15 @@ def create_design_model(sender, instance: Job, **kwargs):  # pylint:disable=unus
         instance (Job): Job instance that has been created or updated.
     """
     if instance.job_class and issubclass(instance.job_class, DesignJob):
-        _, created = Design.objects.get_or_create(job=instance)
+        default_data = {}
+        if hasattr(instance.job_class.Meta, "description"):
+            default_data["description"] = instance.job_class.Meta.description
+        if hasattr(instance.job_class.Meta, "version"):
+            default_data["version"] = instance.job_class.Meta.version
+        if hasattr(instance.job_class.Meta, "docs"):
+            default_data["docs"] = instance.job_class.Meta.docs
```

Comment:
This is an additional comment based on my previous question about storing these values in the database. If we have them in the class why store them in the database? Which is authoritative? If the design class description is updated should the database get updated? Also, do we want to allow people to override the values via the UI?

Reply:
it makes sense. Moving it to properties